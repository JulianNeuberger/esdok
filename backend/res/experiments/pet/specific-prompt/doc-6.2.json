{"text": "The process starts periodically on the first of each month , when Assembler AG places an order with the supplier in order to request more product parts . a ) Assembler AG sends the order to the supplier . b ) The supplier processes the order . c ) The supplier sends an invoice to Assembler AG . d ) Assembler AG receives the invoice .", "name": "doc-6.2", "id": "doc-6.2", "category": "", "tokens": [{"text": "The", "indexInDocument": 0, "posTag": "DT", "sentenceIndex": 0}, {"text": "process", "indexInDocument": 1, "posTag": "NN", "sentenceIndex": 0}, {"text": "starts", "indexInDocument": 2, "posTag": "VBZ", "sentenceIndex": 0}, {"text": "periodically", "indexInDocument": 3, "posTag": "RB", "sentenceIndex": 0}, {"text": "on", "indexInDocument": 4, "posTag": "IN", "sentenceIndex": 0}, {"text": "the", "indexInDocument": 5, "posTag": "DT", "sentenceIndex": 0}, {"text": "first", "indexInDocument": 6, "posTag": "JJ", "sentenceIndex": 0}, {"text": "of", "indexInDocument": 7, "posTag": "IN", "sentenceIndex": 0}, {"text": "each", "indexInDocument": 8, "posTag": "DT", "sentenceIndex": 0}, {"text": "month", "indexInDocument": 9, "posTag": "NN", "sentenceIndex": 0}, {"text": ",", "indexInDocument": 10, "posTag": ",", "sentenceIndex": 0}, {"text": "when", "indexInDocument": 11, "posTag": "WRB", "sentenceIndex": 0}, {"text": "Assembler", "indexInDocument": 12, "posTag": "NNP", "sentenceIndex": 0}, {"text": "AG", "indexInDocument": 13, "posTag": "NNP", "sentenceIndex": 0}, {"text": "places", "indexInDocument": 14, "posTag": "VBZ", "sentenceIndex": 0}, {"text": "an", "indexInDocument": 15, "posTag": "DT", "sentenceIndex": 0}, {"text": "order", "indexInDocument": 16, "posTag": "NN", "sentenceIndex": 0}, {"text": "with", "indexInDocument": 17, "posTag": "IN", "sentenceIndex": 0}, {"text": "the", "indexInDocument": 18, "posTag": "DT", "sentenceIndex": 0}, {"text": "supplier", "indexInDocument": 19, "posTag": "NN", "sentenceIndex": 0}, {"text": "in", "indexInDocument": 20, "posTag": "IN", "sentenceIndex": 0}, {"text": "order", "indexInDocument": 21, "posTag": "NN", "sentenceIndex": 0}, {"text": "to", "indexInDocument": 22, "posTag": "TO", "sentenceIndex": 0}, {"text": "request", "indexInDocument": 23, "posTag": "VB", "sentenceIndex": 0}, {"text": "more", "indexInDocument": 24, "posTag": "JJR", "sentenceIndex": 0}, {"text": "product", "indexInDocument": 25, "posTag": "NN", "sentenceIndex": 0}, {"text": "parts", "indexInDocument": 26, "posTag": "NNS", "sentenceIndex": 0}, {"text": ".", "indexInDocument": 27, "posTag": ".", "sentenceIndex": 0}, {"text": "a", "indexInDocument": 28, "posTag": "DT", "sentenceIndex": 1}, {"text": ")", "indexInDocument": 29, "posTag": ")", "sentenceIndex": 1}, {"text": "Assembler", "indexInDocument": 30, "posTag": "NNP", "sentenceIndex": 1}, {"text": "AG", "indexInDocument": 31, "posTag": "NNP", "sentenceIndex": 1}, {"text": "sends", "indexInDocument": 32, "posTag": "VBZ", "sentenceIndex": 1}, {"text": "the", "indexInDocument": 33, "posTag": "DT", "sentenceIndex": 1}, {"text": "order", "indexInDocument": 34, "posTag": "NN", "sentenceIndex": 1}, {"text": "to", "indexInDocument": 35, "posTag": "TO", "sentenceIndex": 1}, {"text": "the", "indexInDocument": 36, "posTag": "DT", "sentenceIndex": 1}, {"text": "supplier", "indexInDocument": 37, "posTag": "NN", "sentenceIndex": 1}, {"text": ".", "indexInDocument": 38, "posTag": ".", "sentenceIndex": 1}, {"text": "b", "indexInDocument": 39, "posTag": "NN", "sentenceIndex": 2}, {"text": ")", "indexInDocument": 40, "posTag": ")", "sentenceIndex": 2}, {"text": "The", "indexInDocument": 41, "posTag": "DT", "sentenceIndex": 2}, {"text": "supplier", "indexInDocument": 42, "posTag": "NN", "sentenceIndex": 2}, {"text": "processes", "indexInDocument": 43, "posTag": "VBZ", "sentenceIndex": 2}, {"text": "the", "indexInDocument": 44, "posTag": "DT", "sentenceIndex": 2}, {"text": "order", "indexInDocument": 45, "posTag": "NN", "sentenceIndex": 2}, {"text": ".", "indexInDocument": 46, "posTag": ".", "sentenceIndex": 2}, {"text": "c", "indexInDocument": 47, "posTag": "NN", "sentenceIndex": 3}, {"text": ")", "indexInDocument": 48, "posTag": ")", "sentenceIndex": 3}, {"text": "The", "indexInDocument": 49, "posTag": "DT", "sentenceIndex": 3}, {"text": "supplier", "indexInDocument": 50, "posTag": "NN", "sentenceIndex": 3}, {"text": "sends", "indexInDocument": 51, "posTag": "VBZ", "sentenceIndex": 3}, {"text": "an", "indexInDocument": 52, "posTag": "DT", "sentenceIndex": 3}, {"text": "invoice", "indexInDocument": 53, "posTag": "NN", "sentenceIndex": 3}, {"text": "to", "indexInDocument": 54, "posTag": "TO", "sentenceIndex": 3}, {"text": "Assembler", "indexInDocument": 55, "posTag": "NNP", "sentenceIndex": 3}, {"text": "AG", "indexInDocument": 56, "posTag": "NNP", "sentenceIndex": 3}, {"text": ".", "indexInDocument": 57, "posTag": ".", "sentenceIndex": 3}, {"text": "d", "indexInDocument": 58, "posTag": "NN", "sentenceIndex": 4}, {"text": ")", "indexInDocument": 59, "posTag": ")", "sentenceIndex": 4}, {"text": "Assembler", "indexInDocument": 60, "posTag": "NNP", "sentenceIndex": 4}, {"text": "AG", "indexInDocument": 61, "posTag": "NNP", "sentenceIndex": 4}, {"text": "receives", "indexInDocument": 62, "posTag": "VBZ", "sentenceIndex": 4}, {"text": "the", "indexInDocument": 63, "posTag": "DT", "sentenceIndex": 4}, {"text": "invoice", "indexInDocument": 64, "posTag": "NN", "sentenceIndex": 4}, {"text": ".", "indexInDocument": 65, "posTag": ".", "sentenceIndex": 4}], "mentions": [{"type": "activity", "tokenDocumentIndices": [14, 15, 16]}, {"type": "activity", "tokenDocumentIndices": [23]}, {"type": "activity", "tokenDocumentIndices": [32, 33, 34]}, {"type": "activity", "tokenDocumentIndices": [43, 44, 45]}, {"type": "activity", "tokenDocumentIndices": [51, 52, 53]}, {"type": "activity", "tokenDocumentIndices": [62, 63, 64]}, {"type": "actor", "tokenDocumentIndices": [12, 13]}, {"type": "actor", "tokenDocumentIndices": [18, 19]}, {"type": "actor", "tokenDocumentIndices": [30, 31]}, {"type": "actor", "tokenDocumentIndices": [41, 42]}, {"type": "actor", "tokenDocumentIndices": [49, 50]}, {"type": "actor", "tokenDocumentIndices": [60, 61]}, {"type": "activity data", "tokenDocumentIndices": [24, 25, 26]}, {"type": "activity data", "tokenDocumentIndices": [33, 34]}, {"type": "activity data", "tokenDocumentIndices": [44, 45]}, {"type": "activity data", "tokenDocumentIndices": [52, 53]}, {"type": "activity data", "tokenDocumentIndices": [63, 64]}, {"type": "further specification", "tokenDocumentIndices": [17, 18, 19]}, {"type": "further specification", "tokenDocumentIndices": [20, 21, 22, 23, 24, 25, 26]}], "entities": [{"mentionIndices": [6, 8, 11]}, {"mentionIndices": [7, 9, 10]}, {"mentionIndices": [13, 14]}, {"mentionIndices": [15, 16]}, {"mentionIndices": [0]}, {"mentionIndices": [1]}, {"mentionIndices": [2]}, {"mentionIndices": [3]}, {"mentionIndices": [4]}, {"mentionIndices": [5]}, {"mentionIndices": [12]}, {"mentionIndices": [17]}, {"mentionIndices": [18]}], "relations": [{"headMentionIndex": 0, "tailMentionIndex": 1, "type": "flow"}, {"headMentionIndex": 1, "tailMentionIndex": 2, "type": "flow"}, {"headMentionIndex": 2, "tailMentionIndex": 3, "type": "flow"}, {"headMentionIndex": 3, "tailMentionIndex": 4, "type": "flow"}, {"headMentionIndex": 4, "tailMentionIndex": 5, "type": "flow"}, {"headMentionIndex": 0, "tailMentionIndex": 6, "type": "actor performer"}, {"headMentionIndex": 0, "tailMentionIndex": 17, "type": "further specification"}, {"headMentionIndex": 0, "tailMentionIndex": 18, "type": "further specification"}, {"headMentionIndex": 1, "tailMentionIndex": 6, "type": "actor performer"}, {"headMentionIndex": 1, "tailMentionIndex": 12, "type": "uses"}, {"headMentionIndex": 2, "tailMentionIndex": 8, "type": "actor performer"}, {"headMentionIndex": 2, "tailMentionIndex": 13, "type": "uses"}, {"headMentionIndex": 2, "tailMentionIndex": 7, "type": "actor recipient"}, {"headMentionIndex": 3, "tailMentionIndex": 9, "type": "actor performer"}, {"headMentionIndex": 3, "tailMentionIndex": 14, "type": "uses"}, {"headMentionIndex": 4, "tailMentionIndex": 10, "type": "actor performer"}, {"headMentionIndex": 4, "tailMentionIndex": 15, "type": "uses"}, {"headMentionIndex": 4, "tailMentionIndex": 8, "type": "actor recipient"}, {"headMentionIndex": 5, "tailMentionIndex": 11, "type": "actor performer"}, {"headMentionIndex": 5, "tailMentionIndex": 16, "type": "uses"}]}