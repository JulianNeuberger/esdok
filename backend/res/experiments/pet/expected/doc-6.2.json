{"nodes": [{"id": "0", "name": "places", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, {"id": "1", "name": "an order", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, {"id": "2", "name": "the supplier", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, {"id": "3", "name": "in order to request more product parts", "entity": {"name": "further specification", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, {"id": "4", "name": "Assembler AG", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, {"id": "5", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, {"id": "6", "name": "processes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, {"id": "7", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, {"id": "8", "name": "an invoice", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, {"id": "9", "name": "receives", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}], "edges": [{"id": "0", "source": {"id": "0", "name": "places", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "1", "name": "an order", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "1", "source": {"id": "0", "name": "places", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "the supplier", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "type": "actor recipient"}, {"id": "2", "source": {"id": "0", "name": "places", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "5", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "3", "source": {"id": "0", "name": "places", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "3", "name": "in order to request more product parts", "entity": {"name": "further specification", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "type": "further specification"}, {"id": "4", "source": {"id": "5", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "1", "name": "an order", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "5", "source": {"id": "5", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "4", "name": "Assembler AG", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "6", "source": {"id": "5", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "the supplier", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "type": "actor recipient"}, {"id": "7", "source": {"id": "5", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "6", "name": "processes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "8", "source": {"id": "6", "name": "processes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "1", "name": "an order", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "9", "source": {"id": "6", "name": "processes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "the supplier", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "10", "source": {"id": "6", "name": "processes", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "7", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "11", "source": {"id": "7", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "8", "name": "an invoice", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "12", "source": {"id": "7", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "4", "name": "Assembler AG", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "type": "actor recipient"}, {"id": "13", "source": {"id": "7", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "2", "name": "the supplier", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "type": "actor performer"}, {"id": "14", "source": {"id": "7", "name": "sends", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "9", "name": "receives", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "type": "flow"}, {"id": "15", "source": {"id": "9", "name": "receives", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "8", "name": "an invoice", "entity": {"name": "activity data", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "type": "uses"}, {"id": "16", "source": {"id": "9", "name": "receives", "entity": {"name": "activity", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "target": {"id": "4", "name": "Assembler AG", "entity": {"name": "actor", "description": "", "aspect": {"name": "", "textColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shapeColor": {"r": 0, "g": 0, "b": 0, "hex": "#000000"}, "shape": "rectangle"}, "position": {"x": 0, "y": 0}}, "position": {"x": 0, "y": 0}, "source": {"file": "doc-6.2", "pageStart": 1, "pageEnd": 1}}, "type": "actor recipient"}]}